src/DotNetty.Transport/Channels/AbstractChannel.cs

```diff
@@ -366,7 +366,7 @@ internal IMessageSizeEstimatorHandle EstimatorHandle
         /// <summary>
         /// {@link Unsafe} implementation which sub-classes must extend and use.
         /// </summary>
-        protected abstract class AbstractUnsafe : IChannelUnsafe
+        public abstract class AbstractUnsafe : IChannelUnsafe
```

Comment:
Needed to add a derived implementation for `DefaultUnsafe` inside `EmbeddedChannel`